.github/workflows/plugins_react_tests.yml

```
@@ -65,12 +68,26 @@ jobs:
         with:
           repository: ${{ matrix.plugin }}
           path: ${{ github.workspace }}/projects/plugin
+      - name: store plugin name
+        run: echo "PLUGIN_NAME=$(echo ${{ matrix.plugin }} | awk -F'/' '{print $NF}')" >> "${GITHUB_ENV}"
+      - name: Set up plugin in Foreman
+        run: |
+          echo "gemspec name: '$PLUGIN_NAME', path: '${{ github.workspace }}/projects/plugin'" > "bundler.d/$PLUGIN_NAME.local.rb"
+          if [ -d $PLUGIN_NAME/gemfile.d ] ; then
+            cat $PLUGIN_NAME/gemfile.d/*.rb >> bundler.d/$PLUGIN_NAME.local.rb
+          fi
+        working-directory: ${{ github.workspace }}/projects/foreman
       - name: Generate ${{ matrix.plugin }} npm dependencies package-lock
         run: npm install --package-lock-only --no-audit --legacy-peer-deps
         working-directory: ${{ github.workspace }}/projects/plugin
       - name: Install ${{ matrix.plugin }} npm dependencies
         run: npm ci --no-audit --legacy-peer-deps
         working-directory: ${{ github.workspace }}/projects/plugin
+      - run: sudo apt-get update
+      - run: sudo apt-get -qq -y install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev libvirt-dev
+      - name: Install gems
+        run: bundle install 
+        working-directory: ${{ github.workspace }}/projects/foreman
       - name: Run ${{ matrix.plugin }} tests
-        run: npm test
-        working-directory: ${{ github.workspace }}/projects/plugin
+        run: npm run test:plugins $(echo ${{ matrix.plugin }} | awk -F'/' '{print $NF}')
```

```suggestion
        run: npm run test:plugins $PLUGIN_NAME
```

that's the same, right?